docs/scaling/multi-node-setup.txt

```diff
@@ -111,62 +120,62 @@ for more information.
 Master Node Election
 ====================
 
-Although all CrateDB nodes in a cluster are equal, one node is elected as
-master for managing cluster meta data. Like any other peer-to-peer system,
-nodes communicate with each other directly. The master node is responsible for
-making changes to, publishing the global cluster state, and delegating
-redistribution of shards when nodes join or leave the cluster. All nodes are
-eligible to become master.
+In a CrateDB cluster only one master node is responsible for making changes to the
```

Comment:
"In a CrateDB cluster only one master node " - this implies there are multiple master nodes.

===> "In a CrateDB cluster the master node is responsible ... "